src/service/server.rs

```diff
@@ -19,60 +20,68 @@ use crate::util::error::{quit_error, ErrorHints};
 ///
 /// Spawns a tokio runtime to complete all work on.
 #[tokio::main(flavor = "multi_thread")]
-pub async fn service(config: Arc<Config>) -> Result<(), ()> {
-    // Load server state
-    let server = Arc::new(Server::default());
-
+pub async fn service(bind_addr: SocketAddr, configs: Vec<Arc<Config>>) -> Result<(), ()> {
```

Comment:
I haven't inspected this in-depth right now. I'll try to do that later when the other changes have been made.